pokecli.py

```diff
@@ -208,6 +208,17 @@ def init_config():
         type=int,
         default=50
     )
+
+    add_config(
+        parser,
+        load,
+        short_flag="-n",
+        long_flag="--navigator",
+        help="Set the navigator to be used(DEFAULT spiral)",
+        type=str,
+        default=None
```

Comment:
I believe we want to load the default from the config.json file. And then fallback to 'spiral'. If you follow the pattern the other ones show, you can do it all on this line.